clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListTables.java

```diff
@@ -48,6 +49,10 @@ public ListTables(
   /** List the names of all tables in a schema. */
   @Override
   public void handle() {
+    if (metalake == null || catalog == null || schema == null) {
+      showUnSpecifiedMessage();
+      return;
+    }
```

Comment:
It might be better to add a method to the command base class and call that, as it is likely several commands need this? Just something to consider.

Reply:
> It might be better to add a method to the command base class and call that, as it is likely several commands need this? Just something to consider.

Thank you Justin. Can I modify the base classes freely? e.g. `Command` If so, I could move the argument handling logic into the Command class and update all the subclasses accordingly.

Reply:
> It might be better to add a method to the command base class and call that, as it is likely several commands need this? Just something to consider.

hi Justin, I noticed that the metalake parameter is almost never empty, it is given the information during the parsing phase, so do I need to check this parameter in the `handle` method?